cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -10,10 +10,19 @@ import (
 	"github.com/creachadair/jrpc2"
 
 	"github.com/stellar/go/strkey"
+	"github.com/stellar/go/support/collections/set"
 	"github.com/stellar/go/support/errors"
+	"github.com/stellar/go/support/log"
 	"github.com/stellar/go/xdr"
 
-	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/events"
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/db"
+)
+
+const (
+	LedgerScanLimit     = 8000
+	maxContractIDsLimit = 5
+	maxTopicsLimit      = 5
```

Comment:
5?

Reply:
oh for the list of topic filters, not topics in a single topic filter lol

Reply:
Yeah it is bit confusing but yes it is maximum 5 topics per filter: max 5 topics list --> each list has 4 topic strings.